app/models/rejection.rb

```diff
@@ -37,6 +37,9 @@ class Rejection < ActiveRecord::Base
   validates :reasons, presence: true
   validate :validate_allowance_renews_on_date
 
+  before_save do
```

Comment:
It would be good to get this behaviour for unsaved objects, we could do it by overloading the `reasons` method to parse the db data as a set.

Something like this:

```ruby
def reasons
  Set.new(self[:reasons])
end
```

What do you think?